lib/access-code-manager.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+module.exports = (config, promise, random, storageServiceFactory) => {
+    var codeLength, secretLength, storageService;
+
+    /**
+     * Create a set of API credentials
+     *
+     * @param {string} accountId
+     * @param {accessCodeManager~createRequest} request
```

Comment:
Are you missing the type def for this. I would thin it'd be in this file since this is the Access Code Manager.